File: 0027.移除元素/0027-移除元素.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        num=0
        for i in range(len(nums)):
            if(nums[i]==val):
                num+=1
        for n in range(num):
            nums.remove(val)
        return len(nums)

        

File: 0027.移除元素/0027-移除元素.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int slow = 0; 
        int fast = 0;
        for( int i = 0; i < nums.size(); i++){
            if( nums[i] != val){
                nums[slow] = nums[fast];
                slow++;
            }
            fast++;
        }
        return slow;
        // int slow = 0;
        // for( int fast = 0; fast < nums.size(); fast++){
        //     if( nums[fast] != val){
        //         nums[slow] = nums[fast];
        //         slow+=1;
        //     }
            
        // }
        // return slow;

    }
};